views/partials/content-sermon-wrapper-start.php

```diff
@@ -52,6 +52,14 @@
 	case 'oceanwp':
 		echo '<div id="content-wrap" class="container clr"><div id="primary" class="content-area clr"><div id="content" class="site-content clr">';
 		break;
+  case 'zerif-lite':
+		echo '</header>';
```

Comment:
Weird that the the header is not already closed, gotta check this one.

Reply:
Seems to be a Bug of Zerif Lite, as my installation was missing that tag.